wg-deployment/charter.md

```diff
@@ -0,0 +1,35 @@
+# WG Deployment Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+Manage manifests for deploying and operating Kubeflow.
+Develop and maintain tools like kfctl (CLI + operator) for using the manifests to deploy Kubeflow.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- kfctl
+- kfdef for each platform
+
+#### Cross-cutting and Externally Facing Processes
+
+- Cutting releases on both the manifests and kfctl repos
+- Qualifying a Kubeflow release for each platform
```

Comment:
I would think qualifying Kubeflow releases for each platform would be the platform owners responsibility?